src/dbus_fast/message_bus.py

```diff
@@ -892,8 +922,11 @@ def _make_method_handler(
         def _callback_method_handler(
             msg: Message, send_reply: Callable[[Message], None]
         ) -> None:
+            result = method_fn(interface, *msg_body_to_args(msg))
+            if not _expects_reply(msg):
+                return
```

Comment:
The default was to always call send_reply when a reply wasn't expected it would just be thrown away. All that work for 6500 DeviceFound messages per minute and they are just discarded 🙈